physics/sfcsub.F

```diff
@@ -1902,44 +1842,20 @@ subroutine sfccycle(lugb,len,lsoil,sig1t,deltsfc                  &
      &                siclmx,siclmn,sicomx,sicomn,sicimx,sicimn,
      &                sicjmx,sicjmn,sicsmx,sicsmn,epssic,
      &                rla,rlo,len,kqcm,percrit,lgchek,me)
-          call qcmxmn('smc1f    ',smcfcs(1,1),slifcs,snofcs,icefl1,
-     &                smclmx,smclmn,smcomx,smcomn,smcimx,smcimn,
-     &                smcjmx,smcjmn,smcsmx,smcsmn,epssmc,
-     &                rla,rlo,len,kqcm,percrit,lgchek,me)
-          call qcmxmn('smc2f   ',smcfcs(1,2),slifcs,snofcs,icefl1,
-     &                smclmx,smclmn,smcomx,smcomn,smcimx,smcimn,
-     &                smcjmx,smcjmn,smcsmx,smcsmn,epssmc,
-     &                rla,rlo,len,kqcm,percrit,lgchek,me)
-!clu [+8l] add smcfcs(3:4)
-          if(lsoil.gt.2) then
-          call qcmxmn('smc3f    ',smcfcs(1,3),slifcs,snofcs,icefl1,
-     &                smclmx,smclmn,smcomx,smcomn,smcimx,smcimn,
-     &                smcjmx,smcjmn,smcsmx,smcsmn,epssmc,
-     &                rla,rlo,len,kqcm,percrit,lgchek,me)
-          call qcmxmn('smc4f   ',smcfcs(1,4),slifcs,snofcs,icefl1,
+!-- soil moisture forecast
+        do k=1,lsoil
+          call qcmxmn('smcf    ',smcfcs(1,k),slifcs,snofcs,icefl1,
```

Comment:
Same here.

Reply:
and throughout the rest of the file

Reply:
agree

Reply:
Can you please make these changes everywhere?